docs/bokeh/source/docs/user_guide/topics/categorical.rst

```diff
@@ -107,6 +107,24 @@ in this chapter:
 .. bokeh-plot:: __REPO__/examples/topics/categorical/periodic.py
     :source-position: below
 
+When we have more than three to four quantitative variables, it is sometimes more useful
+to quantify the amount of association between pairs of variables and visualize this quantity
+rather than the raw data. One common way to do this is to calculate correlation coefficients.
+Visualizations of correlation coefficients are called correlograms.
+
+The following correlogam is another good example of the techniques in this chapter.
+
+In this plot, we display the correlations as colored circles and scale the circle size with
+the absolute value of the correlation coefficient. In this way, low correlations are suppressed
+and high correlations stand out better.
+
+This example uses the ``LinearColorMapper`` to map the colors of the plot
+in order to highlight the correlations between the pair of elements. This mapper is also
+passed to the color bar to provide a visual legend below:
+
+.. bokeh-plot:: _REPO_/examples/topics/categorical/correlograms.py
```

Comment:
I think after changing the filename, you also need to adjust this:
```suggestion
.. bokeh-plot:: _REPO_/examples/topics/categorical/correlogram.py
```

Reply:
I'km going to go ahead and commit this and then merge in order to get this over the line before monday